packages/tupaia-web/src/views/ProjectsModal.tsx

```diff
@@ -53,20 +61,38 @@ const ExploreButton = styled(RouterButton).attrs({
   color: 'default',
   to: DEFAULT_URL,
 })`
-  margin-bottom: 16px;
-  width: 250px;
-  height: 50px;
+  margin-top: 0.3rem;
+  margin-bottom: 1rem;
+  margin-left: 0.3rem;
+  width: 10.5rem;
+  height: 2.5rem;
+  line-height: 1.125rem;
   border-radius: 3px;
-  font-size: 13px;
+  font-size: 0.875rem;
+  font-weight: 500;
+  font-style: normal;
+  text-align: center;
+  text-transform: none;
+  border-color: white;
+`;
 
-  svg {
-    margin-right: 10px;
-  }
+const Line = styled.div`
+  background-color: #9ba0a6;
+  height: 1px;
+  margin-top: 0.7rem;
+`;
+
+const ProjectsTitle = styled.h1`
```

Comment:
Can we change this to use `Typography` (you can set `attrs` on it to tell it to be an h1` from material ui, and use the `theme.palette.text.primary` value for the colour?